src/linear_algebra/matrix/reindex.lean

```diff
@@ -52,14 +57,65 @@ rfl
   reindex_linear_equiv (equiv.refl m) (equiv.refl n) = linear_equiv.refl R _ :=
 linear_equiv.ext $ λ _, rfl
 
-lemma reindex_linear_equiv_mul {o o' : Type*} [fintype o] [fintype o'] [semiring R]
+lemma reindex_linear_equiv_trans [semiring R] (e₁ : m ≃ m') (e₂ : n ≃ n') (e₁' : m' ≃ m'')
+  (e₂' : n' ≃ n'') : (reindex_linear_equiv e₁ e₂).trans (reindex_linear_equiv e₁' e₂') =
+   @reindex_linear_equiv _ _ _ _ _ _ _ _ R _ (e₁.trans e₁') (e₂.trans e₂') :=
```

Comment:
Can you replace these underscores with a type ascription?

Reply:
I don't know what you mean by "type ascription"...

Reply:
I mean `(sorry : _ ≃ₗ[R] _)` like in the symm proof above

Reply:
Type ascription does not seem to be of much use, I was unable to make it work. On the other hand, I'll try to work along the lines of what you suggested of making stronger use of your previous `PR` on `reindex`.

Reply:
I'll have a go at this later

Reply:
The obvious fix appears to work just fine:
```suggestion
   (reindex_linear_equiv (e₁.trans e₁') (e₂.trans e₂') : _ ≃ₗ[R] _) :=
```